src/widgets/settings_widget.h

```diff
@@ -30,6 +30,7 @@ enum GameOptionsWidgets {
 	WID_GO_BASE_SFX_DESCRIPTION = WID_GO_BASE_SFX_TEXTFILE + TFT_END,     ///< Description of selected base SFX.
 	WID_GO_BASE_MUSIC_DROPDOWN,    ///< Use to select a base music set.
 	WID_GO_BASE_MUSIC_VOLUME,      ///< Change music volume.
+	WID_GO_BASE_MUSIC_OPEN_JUKEBOX,///< Open the jukebox.
```

Comment:
Missing space here is irritating, as it requires realigning all the others... perhaps `WID_GO_BASE_MUSIC_JUKEBOX` would be sufficient? (drop the 'OPEN')